[WIP]ユーザー新規登録/ログインページ

Ken-Yamamoto opened the pull request:
#WHAT
・ユーザー新規登録、login画面のコーディング
#WHY
・ユーザー新規登録、login時に必要な画面のため

Ken-Yamamoto committed:
Task7_signUp&signIn

#WHAT
新規登録1page目の先行確認のため
#WHY
部分的に完了したため

Ken-Yamamoto commented:
sign-up 1page目のみ、できているところまで先行確認をお願い致します。
・作成分
[![Image from Gyazo](https://i.gyazo.com/d63c47e07e2484b2e681ba75e462f11c.png)](https://gyazo.com/d63c47e07e2484b2e681ba75e462f11c)
・本家分
[![Image from Gyazo](https://i.gyazo.com/1caaadc732aa448ced1344488c3371ba.png)](https://gyazo.com/1caaadc732aa448ced1344488c3371ba)

yuki-uchida commented on app/assets/stylesheets/signUp.scss:
```diff
@@ -0,0 +1,120 @@
+}
+
+.sign-up-main{
+  .sign-up-main-upper{
```
アンバサンド使いましょう

yuki-uchida commented on app/assets/stylesheets/signUp.scss:
```diff
@@ -0,0 +1,120 @@
+      font-size: 16px;
+    }
+    .btn-facebook{
+      position: relative;
```
BEMに従ってクラス命名をしましょう

Ken-Yamamoto replied:
.sign-up-center__content__btnを基準として--mail,--facebook,--googleとして命名修正しました。

yuki-uchida commented on app/views/products/signUp.html.haml:
```diff
@@ -0,0 +1,37 @@
+            =link_to'特定商取引に関する表記','https://www.mercari.com/jp/tokutei/'
+      .devise-footer__logo
+        =link_to'https://www.mercari.com/jp/' do
+          %img{src:'//www-mercari-jp.akamaized.net/assets/img/common/common/logo-gray.svg?570108281',alt:'mercari', style:'width:80px;', style:'height:65px;'} 
```
image_tagというヘルパーメソッドがあるのでそちらを使いましょう

Ken-Yamamoto replied:
image_tag使用へ修正完了です。

yuki-uchida replied:
push忘れてないですか？

yuki-uchida commented on app/assets/stylesheets/signUp.scss:
```diff
@@ -0,0 +1,120 @@
+.devise-container{
+  padding: 0 0 220px;
+  text-align: center;
+  .devise-header{
```
BEMに従いましょう

Ken-Yamamoto replied:
.devise-container__headerへ修正しました。

Ken-Yamamoto replied:
各所変更後のviewとなります。
[![Image from Gyazo](https://i.gyazo.com/282bbcd128895247f98f1f3d2500678e.png)](https://gyazo.com/282bbcd128895247f98f1f3d2500678e)

Ken-Yamamoto committed:
Edit files related on signUp

BEMに則り命名修正、scssファイルをアンパサンド使用にて一部修正
image_tag未使用箇所を使用へ修正
signUp.html.haml
signUp.scss
deviseFooter.scss

Ken-Yamamoto committed:
Edit signUp.html.haml

image_tag反映のため

hiroki5123 commented on app/views/products/signUp.html.haml:
```diff
@@ -0,0 +1,38 @@
+                = fa_icon 'envelope', class: 'icon' 
+                メールアドレスで登録
+              .sign-up-center__content__btn--facebook
+                = image_tag 'flogo-HexRBG-Wht-58.png', class: 'icon',size: "20x20"
```
大きさはCSS側に指定しましょう

hiroki5123 commented on app/views/products/signUp.html.haml:
```diff
@@ -0,0 +1,38 @@
+                = image_tag 'flogo-HexRBG-Wht-58.png', class: 'icon',size: "20x20"
+                Facebookで登録 
+              .sign-up-center__content__btn--google
+                = image_tag 'google-logo-web.png', class: 'icon',size: "20x20"
```
大きさはCSS側に指定しましょう

Ken-Yamamoto committed:
Modify signUp.html.haml

Logoの直書きcssをscssファイルへ記載

Ken-Yamamoto commented:
scssファイルへの記載完了です。

yuki-uchida reviewed (approved):
LGTMです。お疲れ様でした
